docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  fdk-model-publisher:
+    build: .
+    image: eu.gcr.io/digdir-fdk-infra/fdk-model-publisher:test
```

Comment:
Image er vel ikke nødvendig hvis man har build

Reply:
rimelig sikker på at det er dette som gjør imaget tagges med det navnet, navnet brukes i løpet av deploy

@stigbd er det korrekt?

Reply:
If you specify image as well as build, then Compose names the built image with the "eu.gcr.io/digdir-fdk-infra/fdk-model-publisher" and optional tag specified in image "test"

Ikke relevant hva som brukes i deploy for denne filen, men ok.

Reply:
Ikke relevant til hva som brukes i deploy?